smartsim/_core/control/controller.py

```diff
@@ -95,6 +96,7 @@ def __init__(self, launcher: str = "local") -> None:
         self._jobs = JobManager(JM_LOCK)
         self.init_launcher(launcher)
         self._telemetry_monitor: t.Optional[subprocess.Popen[bytes]] = None
+        self._exp_id = _helpers.create_short_id_str()
```

Comment:
Do you think that giving experiments a (short) uuid (as we will do with entities) would help? I am starting to think that we may need some sort of `SmartSimBase` object which has basic traits we may reuse in different contexts.

Reply:
I think we're likely to need that for all entities and am behind the idea. I'll create a follow-up ticket to propose it to the group

Reply:
WRT the naming... I can be persuaded that it's misleading here, being called `_exp_id` and existing on the controller. My thoughts when creating it to justify it are: 

- the controller isn't really "surfaced" to a user so I don't call it `ctrl_id`
- the controller is a 1-1 relationship w/the experiment
- given the tight coupling, the controller serves as a fine proxy to the exp
- I avoided also publishing/changing the experiment class for a fix I don't really like (this was for multi-experiment drivers to work seamlessly in the dashboard)

Reply:
removed. see commit `remove "multi-exp driver" support (exp_id)`